bgstally/windows/fleetcarrier.py

```diff
@@ -54,8 +54,18 @@ def show(self):
 
         current_row = 0
 
-        ttk.Label(frm_items, text=_("Selling Materials"), font=FONT_HEADING_2).grid(row=current_row, column=0, sticky=tk.W) # LANG: Label on carrier window
-        ttk.Label(frm_items, text=_("Buying Materials"), font=FONT_HEADING_2).grid(row=current_row, column=1, sticky=tk.W) # LANG: Label on carrier window
+        #ttk.Checkbutton(frm_discordoptions, text=_("Abbreviate Faction Names"), variable=self.bgstally.state.AbbreviateFactionNames, onvalue=CheckStates.STATE_ON, offvalue=CheckStates.STATE_OFF, command=partial(self._option_change, activity)).grid(row=current_row, column=0, padx=10, sticky=tk.W); current_row += 1 # LANG: Checkbox label
+        dummy = ""
+        a = ""
+        b = ""
+        s = ttk.Style() # LANG: Checkbox label
```

Comment:
1. Add type hint
2. Remove `# LANG:` comment from this line - that is additional info for translators when there is a translatable string on the same line, but there is no translatable string on this line.